deployments/datahub/images/default/Dockerfile

```diff
@@ -242,6 +242,9 @@ ENV NLTK_DATA ${CONDA_DIR}/nltk_data
 COPY connectors/text.bash /usr/local/sbin/connector-text.bash
 RUN /usr/local/sbin/connector-text.bash
 
+COPY connectors/2019-fall-phys-188-288.bash /usr/local/sbin/
```

Comment:
The file seems to be missing from the PR?

Reply:
@yuvipanda aah got it. How to attach this file as part of PR?

Reply:
@balajialg the file was removed.

However, you can view what it used to be by following the URL given in the request:
https://github.com/berkeley-dsep-infra/datahub/commit/942b4c8c208e380cc288bbdd59b06ab06d964cad#diff-2cb9238ae53e266b72f1de1268e9818a2ee79801c9030f9eaa3eb9b2be785c6f

Then clicking on the 3 dots next to the deleted file to get access to former contents:
https://github.com/berkeley-dsep-infra/datahub/blob/1e2f491f49f08bde1a34e41c98cb22f1b86b733b/deployments/datahub/images/default/connectors/2019-fall-phys-188-288.bash

So in this case it'd probably be easiest to just create a new file in the location indicated, paste the contents there into it, then git add it, git commit it, then git push it.

Reply:
@felder Thanks a lot for the inputs. Made the corresponding changes and pushed them as part of the same PR